k8s/alasybil.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             - name: HYDRA_DELEGATED_ROUTING_ADDR
               value: "http://127.0.0.1:9999/" # must be a valid URL
             - name: HYDRA_DELEGATED_ROUTING_TIMEOUT
-              value: "2"
+              value: "1000"
```

Comment:
just a side note, these k8s files are unused and will probably be removed once @thattommyhall finishes moving the hydra infra code to a public repo

Reply:
there is a code change from 'seconds' to 'milliseconds' as the units here that's worth keeping in sync across wherever these env vars do get set :)